client/galaxy/scripts/mvc/tours.js

```diff
@@ -32,10 +32,15 @@ define(['libs/bootstrap-tour'],function(BootstrapTour) {
             }
             if (step.postclick){
                 step.onHide = function(){
-                    _.each(step.postclick, function(postclick){
```

Comment:
This is a good idea.  That said, it looks to me like step.onNext should completely replace the parent step.onHide block, otherwise it looks like what would happen here is that onHide, we'd actually be attaching new onNext functions every time, right?